public/index.php

```diff
@@ -16,6 +16,14 @@
 	));
 });
 
+$klein->respond('/create', function() use ($twig, $crud){
```

Comment:
please checkout Klein documentation on how to properly define routes with methods
https://github.com/klein/klein.php

do the same for all routes, not just this one

Reply:
Something like this for all routes?
$klein->respond('GET','/', function() use ($twig, $crud){
	$getAll = $crud->getAll();
	echo $twig->render('home.twig', array(
		'getAll' => $getAll
	));
});

Reply:
that is correct, for this case - method name needs to correspond to the action you want to achieve 

here is a nice summation of most commonly used methods:
http://www.restapitutorial.com/lessons/httpmethods.html